content/en/serverless/google_cloud_run/functions_gen1.md

```diff
@@ -69,12 +71,65 @@ This page is **only for legacy 1st Gen Cloud Run Functions**, for Gen 2 support,
 [2]: /tracing/metrics/runtime_metrics/?tab=python
 [3]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=python
 {{< /programming-lang >}}
+{{< programming-lang lang="java" >}}
+1. **Install dependencies**. Run the following commands in the terminal of your source directory:
+   ```shell
+   wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
+   wget -O dd-serverless-compat-java-agent.jar 'https://dtdg.co/latest-serverless-compat-java-agent'
+   ```
+   you can also install the tracer using the following maven dependency:
+   ```xml
+   <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-antrun-plugin</artifactId>
+       <version>1.8</version>
+       <executions>
+           <execution>
+               <phase>initialize</phase>
+               <configuration>
+                   <tasks>
+                       <get src="https://dtdg.co/latest-java-tracer" dest="dd-java-agent.jar" />
+                        <get src="https://dtdg.co/latest-serverless-compat-java-agent" dest="dd-serverless-compat-java-agent.jar" />
+                   </tasks>
+               </configuration>
+               <goals>
+                   <goal>run</goal>
+               </goals>
+           </execution>
+       </executions>
+   </plugin>
+   ```
+
+   Datadog recommends using the latest versions of both `datadog-serverless-compat` and `ddtrace` to ensure you have access to enhancements and bug fixes.
+
+
+2. **Initialize the Datadog Java tracer and serverless compatibility layer**. Add `JAVA_TOOL_OPTIONS` to your runtime environment variable:
+   
+   [Auto instruments][1], the java tracer by setting the Runtime environment variable to wrap your Java cloud function with the Datadog Java tracer and the serverless compatibility layer.
+
+   | Name      | Value |
+   |-----------| ----- |
+   | `JAVA_TOOL_OPTIONS` | `-javaagent:dd-serverless-compat-java-agent.jar -javaagent:dd-java-agent.jar ` |
```

Comment:
Should we be assuming that customers are placing these jars in the root directory? Maybe we can mimic the instructions in the Java Tracer installation doc since it doesn't make assumptions about where the jars are located.

https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/java/?tab=wget
```
-javaagent:/path/to/dd-java-agent.jar
```